src/buffer/out/TextAttribute.hpp

```diff
@@ -202,6 +225,7 @@ class TextAttribute final
     TextColor _foreground; // sizeof: 4, alignof: 1
     TextColor _background; // sizeof: 4, alignof: 1
     TextColor _underlineColor; // sizeof: 4, alignof: 1
+    MarkKind _markKind; // sizeof: 2, cause I guess a uint8_t is 2B?

```

Comment:
```suggestion
    MarkKind _markKind; // sizeof: 2, alignof: 1
```

since you already made it a `uint16_t`, the comment about it being a u8 is spurious